vaccine_feed_ingest/runners/pa/arcgis/normalize.py

```diff
@@ -28,7 +28,33 @@ def _get_id(site: dict) -> str:
     arcgis = "5b874ac0947347e9be49f6847eb44604"
     layer = 0
 
-    return f"{runner}:{site_name}:{arcgis}_{layer}:{data_id}"
+    return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
+
+
+def _get_website(site: dict) -> Optional[schema.Contact]:
+    if not site["attributes"]["Website"]:
+        return None
+    website_input = site["attributes"]["Website"].strip()
+    # check if the website field contains an email instead, for example
+    # "Social media/home page. Please contact Mktg Director first.last@example.com"
+    email_match = re.search(r"(?P<email>\S+@\S+)", website_input)
+    if email_match:
+        logger.info(f"found email in website: '{website_input}'")
+        return schema.Contact(email=email_match.group("email"))
+
+    # if the input has any spaces after stripping it, this isn't a url we can parse
+    if re.search(r"\s", website_input):
+        raise Exception(f"Unable to parse website: '{website_input}'")
```

Comment:
Does it make sense to log an error and then return `None` so the entire feed isn't blocked on a parsing error?

Reply:
I had log + return None originally, but then I wondered how we'd detect the error. I agree this is a pain to break parsing if we run into new data. Is there any monitoring for error logs in the final process? or something else to detect?